src/AWS.Logger.AspNetCore/NullExternalScopeProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions.Internal;
+using System;
+
+namespace AWS.Logger.AspNetCore
+{
+    /// <summary>
+    /// Scope provider that does nothing.
+    /// </summary>
+    internal class NullExternalScopeProvider : IExternalScopeProvider
+    {
+        private NullExternalScopeProvider()
+        {
+        }
+
+        /// <summary>
+        /// Returns a cached instance of <see cref="NullExternalScopeProvider"/>.
+        /// </summary>
+        public static IExternalScopeProvider Instance { get; } = new NullExternalScopeProvider();
+
+        /// <inheritdoc />
+        void IExternalScopeProvider.ForEachScope<TState>(Action<object, TState> callback, TState state)
```

Comment:
explicit visibility

Reply:
You cannot put a `public` on a explicit interface implementation method because it automatically is. Now maybe the original PR didn't need to do explicit interface implementation but as this is a dummy implementation of an interface I believe the point is to show how this class should not be used for any real purpose for scoping.